### PR TITLE
feat!: remove style rules from eslint for Prettier

### PR DESCRIPTION
**Pull request by westeezy**

### CODE REVIEW

**Review by mstuart — APPROVED** [review]

**Review by gregjopa — APPROVED** [review]
💯

**Review by wsbrunson — APPROVED** [review]
are you setting up the prettier commit-hook and/or running `--check` on CI?

**Comment by westeezy** [comment]
> are you setting up the prettier commit-hook and/or running `--check` on CI?

Yeah I was on the fence about where to put instructions for that step but I think updating the readme and adding it to this repo makes the most sense. I'll  push an update that wires it together in here and adds readme instructions for other repos with pre-existing githooks

**Review by wsbrunson — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -8,10 +8,12 @@
     "lint": "eslint --ext js,jsx config/ test/ *.js",
     "flow-typed": "flow-typed install",
     "flow": "flow",
+    "format": "prettier --write --ignore-unknown .",
+    "format:check": "prettier --check .",
```
Comment (gregjopa) [inline package.json:12]: Thanks for adding these `format` tasks!

**Pull request merged**